src/client/api/SConscript

```diff
@@ -18,7 +18,7 @@ def scons():
     dc_tgts += dc_mgmt_tgts + dc_array_tgts + dc_kv_tgts + dc_security_tgts
     libdaos = daos_build.library(env, 'daos', dc_tgts,
                                  SHLIBVERSION=API_VERSION,
-                                 LIBS=['daos_common'])
+                                 LIBS=['daos_common', 'daos_tests'])
```

Comment:
Do we really want daos_tests to be a dependence of libdaos ?  I think we should either move the helper functions to common or add a new library.

I'll have a look locally and see if I can figure out why you're seeing an issue with the build here though.

Reply:
I'm fine with moving the helpers to common... I put them in daos_tests originally because they're only really intended to be used for tests. But stepping back a bit, I think it's valid to ask whether or not this entire PR is a good idea. On the one hand, it removes a roadblock to getting my new management service implementation landed, but on the other hand it allow technical debt to linger.

Reply:
Based on a skype chat, we'll keep the helpers in daos_test and I'll create a techdebt ticket to remove this linkage once all of the tests have been fixed.